marklogic-client-api/src/main/java/com/marklogic/client/datamovement/QueryBatcher.java

```diff
@@ -380,9 +380,16 @@
   void setMaxUris();
```

Comment:
Did we decide to remove all setters and getter for max uris from the public interface?

Should we have a zero-argument setMaxBatches() that caps the query at the current batch?